src/Classes/ItemsTab.lua

```diff
@@ -1100,6 +1104,187 @@ function ItemsTabClass:Draw(viewPort, inputEvents)
+			end
+		end
+		midPoint = midPoint:sub(1, -2)
+	end
+
+	-- For Unique Items
+	if selItem.rarity == 'UNIQUE' then
+		local removeName = ", " .. selItem.baseName
+		local selName = string.gsub(selItem.name, removeName, "")
+		startPoint = string.format('{"query":{"status":{"option":"online"},"name":"%s","type":"%s","stats":[{"type":"and","filters":[]}],"filters":{}},"sort":{"price":"asc"}}', selName, selItem.baseName)
+		midPoint = ''
+		endPoint = ''
+	end
+
+	local jsonQuery = startPoint..midPoint..endPoint
+	
+	--call the api handler
+	local responeJson = self.TradeAPIRequest(jsonQuery)
+	--get the response
+	local response = self.ProcessJSON(responeJson)
+
+	if not string.match(responeJson, 'error') then	
+		--opens default windows web browser
+		--Only works on Windows
+		--Get current league name? dont worry about it ? unsure. 
+		os.execute(string.format('start https://www.pathofexile.com/trade/search/Scourge/%s', response.id))
+	else
+		main:OpenMessagePopup("Search Failed", "There was an error with this search request.")
+	end
+end 
+
+-- Search for explicit mods
+function ItemsTabClass:SearchExplicitModList(modDesc)
+	local filePath = "Data/StatDescriptions/item_mods.json"
+	local ids = {}
+
+	for i in ipairs(self) do
+		for lines in io.lines(filePath) do
+			if string.find(lines, self[i], 1, true) then
+				local jsonLine = ItemsTabClass.ProcessJSON(lines)
+				table.insert(ids, jsonLine.trade.ids.explicit[1])
+			end
+		end		
+	end
+
+	return ids
+end
+
+-- Search for implicit mods
+function ItemsTabClass:SearchImplicitModList(modDesc)
+	local filePath = "Data/StatDescriptions/item_mods.json"
+	local ids = {}
+
+	for i in ipairs(self) do
+		for lines in io.lines(filePath) do
+			if string.find(lines, self[i], 1, true) then
+				local jsonLine = ItemsTabClass.ProcessJSON(lines)
+				table.insert(ids, jsonLine.trade.ids.implicit[1])
+			end
+		end		
+	end
+
+	return ids
+end
+
+-- Search for fractured mods
+function ItemsTabClass:SearchFracturedModList(modDesc)
+	local filePath = "Data/StatDescriptions/item_mods.json"
+	local ids = {}
+
+	for i in ipairs(self) do
+		for lines in io.lines(filePath) do
+			if string.find(lines, self[i], 1, true) then
+				local jsonLine = ItemsTabClass.ProcessJSON(lines)
+				table.insert(ids, jsonLine.trade.ids.fractured[1])
+			end
+		end		
+	end
+	return ids
+end
+
+-- API request to Poe Trade
+-- I think this would be best to use the users personal Session ID number
+-- This would changed by alter the User-Agent to a Session ID
+function ItemsTabClass:TradeAPIRequest(json_data)
+	local json = self
+
+	local curl = require("lcurl.safe")
+	local page = ""
+	local response = ""
+	local easy = curl.easy()
+	
+	--API Request
+	easy:setopt{
+		url = "https://www.pathofexile.com/api/trade/search/Archnemesis",
+		post = true,
+		httpheader = {'Content-Type: application/json', 'Accept: application/json', 'User-Agent: Path of Building/2.17 (contact: pob@mailbox.org)'},
```

Comment:
I know this is still "WIP" but these values shouldn't be hardcoded.  Currently hard to test because we aren't in Archnemesis league anymore :)